homework9.py

```diff
@@ -0,0 +1,102 @@
+""" Homework9: Functions writing and implementing programs
+in Python """
+
+# Validate
+# Ваша задача написать программу, принимающее число - номер кредитной карты
+# (число может быть четным или не четным).
+# И проверяющей может ли такая карта существовать. Предусмотреть защиту от
+# ввода букв, пустой строки и т.д. Примечания Алгоритм Луна.
+
+from functools import reduce
```

Comment:
move import on the first line of the module, all import should be on the top of the module

Reply:
done)